src/core/main.lua

```diff
@@ -33,23 +33,21 @@ function main ()
    zone("startup")
    require "lib.lua.strict"
    initialize()
+   local program
```

Comment:
@eugeneia probably not. Thing is I don't remember making those changes myself, maybe they come from a merge. Hm...